t/plugin/authz-casbin/policy.csv

```diff
@@ -0,0 +1,3 @@
+p, *, /, GET
+p, admin, *, *
+g, alice, admin
```

Comment:
EOL symbol is required. Please check out the editor configuration.

Reply:
Added the EOL symbol in model/policy files.